src/pages/ProfilDetail.js

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { goBack } from 'react-router-redux';
+import './ProfilDetail.css';
+
+class EventDetail extends PureComponent {
+  render() {
+    const { event, onClose } = this.props;
+
+    return (
+      <div onClick={onClose} className="ProfilDetail-card">
+          <div className="ProfilDetail-close">
+            <div>X</div>
+          </div>
+          <div className="ProfilDetail-monogram"/>
+          <div className="ProfilDetail-content">
+            <div className="ProfilDetail-name">Annette</div>
+            <div className="ProfilDetail-title">You are an enjoyer!</div>
+          </div>
+      </div>
+    );
+  }
+}
+
+const EventDetailContainer = connect(
+  (state, ownProps) => ({
+    event: state.events[ownProps.params.id],
+  }),
+  (dispatch) => ({
+    onClose: () => dispatch(goBack())
```

Comment:
this is good and works as it gives the component a function as prop called onClose that goes back, it would be nice to use the back action from the routing actions defined in actions/routing.js.
Like you did in Overview.js with showProfile